pipeline/test/services/workload-cluster/testUserRbac.sh

```diff
@@ -59,8 +59,8 @@ echo
 echo "Testing user RBAC"
 echo "====================="
 
-user_namespaces=$(yq r "$CONFIG_FILE" 'user.namespaces' | sed 's/-//' - )
-user_admin_users=$(yq r "$CONFIG_FILE" 'user.adminUsers' | sed 's/-//' - )
+user_namespaces=$(yq r "$CONFIG_FILE" 'user.namespaces' | sed 's/-//' - | sed 's/"//g' - )
+user_admin_users=$(yq r "$CONFIG_FILE" 'user.adminUsers' | sed 's/-//' - | sed 's/"//g' - )
```

Comment:
These lines could be simplified a bit, for example
```suggestion
user_admin_users=$(yq r "$CONFIG_FILE" 'user.adminUsers' | tr -d '"-' )
```

Reply:
I wonder if we should try something like [this](https://stackoverflow.com/questions/35005893/convert-a-json-array-to-a-bash-array-of-strings) instead? Otherwise it is just a question of time until someone decides to do this:
```yaml
user:
  adminUsers: ["user1", "user2"]
```
It is perfectly fine to do in yaml, but our parsing here will fail. If we instead get the output from yq as json, we could parse the array as in the linked example.

Reply:
That sounds very much better :slightly_smiling_face:

Reply:
Did some more testing and found that `user.adminUsers[*]` will output the values on separate lines without any `-` or `"` regardless of if it's entered as
```
user:
  adminUsers: ["user1", "user2"]
```
or

```
user:
  adminUsers: 
  - "user1"
  - "user2"
```
So now we should have safe parsing without needing to format it as json first.